code/CMakeLists.txt

```diff
@@ -90,8 +90,8 @@ if(BuildSPEngine OR BuildJK2SPEngine)
 			)
 	else()
 		find_package(SDL2 REQUIRED CONFIG)
-		set(SPEngineIncludeDirectories ${SPEngineIncludeDirectories} ${SDL2_INCLUDE_DIRS})
-		set(SPEngineLibraries ${SPEngineLibraries} ${SDL2_LIBRARIES})
+		set(SPEngineIncludeDirectories ${SPEngineIncludeDirectories})
+		set(SPEngineLibraries ${SPEngineLibraries} SDL2::SDL2)
```

Comment:
Thanks for the PR!

While the syntax is available since CMake 3.0, SDL2 was only updated this March to have the `SDL2::SDL2` target (http://hg.libsdl.org/SDL/diff/ef2d5382a887/sdl2-config.cmake.in). We want to be supporting older versions of SDL2 so we can't accept this change right now unfortunately!